src/test/java/catalinaTema5/DemoqaElements/WebTables.java

```diff
@@ -0,0 +1,46 @@
+package catalinaTema5.DemoqaElements;
+
+import catalinaTema5.BrowserManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class WebTables {
+    public static void main(String[] args) {
+        goToWebTablesPage();
+
+    }
+    public static void goToWebTablesPage() {
+        ChromeDriver driver = BrowserManager.createChromeDriver();
+        driver.get("https://demoqa.com/webtables");
+        WebElement searchButton = driver.findElement(By.id("basic-addon2"));
+        searchButton.click();
+        searchButton.sendKeys("Harry Potter");
+        WebElement AddButton = driver.findElement(By.id("addNewRecordButton"));
```

Comment:
it should be addButton and not AddButton. camelCase in Java when naming variables.